core/CliMulti/RequestCommand.php

```diff
@@ -67,7 +67,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if ($input->getOption('superuser')) {
-            Access::getInstance()->setSuperUserAccess(true);
+            StaticContainer::addDefinitions(array(
+                'observers.global' => \DI\add(array(
+                    array('Environment.bootstrapped', function () {
+                        Access::getInstance()->setSuperUserAccess(true);
+                    })
+                )),
+            ));
```

Comment:
Just to be sure I understand this correctly: the container is already created at that point so *in theory* this wouldn't work. But in this command we include `index.php` so it will re-create a whole environment (and container), so this works. Is that right?